ddl/column.go

```diff
@@ -1102,6 +1099,120 @@ func (w *worker) doModifyColumnTypeWithData(
 	return ver, errors.Trace(err)
 }
 
+func replaceOldColumn(tblInfo *model.TableInfo, oldID, newID int64,
```

Comment:
Why don't we send arguments `oldCol` and `changingCol` directly? Then we needn't to find these two args. And we needn't return `JobStateRollingback`.

Reply:
Because the column info's offset can be incorrect in the case of multi-schema change like `add column first, modify column`.